jackson/src/main/resources/reference.conf

```diff
@@ -1,40 +1,25 @@
 play.modules.enabled += com.lightbend.lagom.javadsl.jackson.JacksonModule
 
-lagom.serialization.json {
-  
-  #//#jackson-modules
-  # The Jackson JSON serializer will register these modules.
-  # It is also possible to use jackson-modules = ["*"] to dynamically
-  # find and register all modules in the classpath.  
-  jackson-modules += "com.fasterxml.jackson.module.paramnames.ParameterNamesModule"
-  jackson-modules += "com.fasterxml.jackson.datatype.jdk8.Jdk8Module"
-  jackson-modules += "com.fasterxml.jackson.datatype.jsr310.JavaTimeModule"
-  jackson-modules += "com.fasterxml.jackson.datatype.pcollections.PCollectionsModule"
-  jackson-modules += "com.fasterxml.jackson.datatype.guava.GuavaModule"
-  #//#jackson-modules  
-  
-  # The serializer will compress the payload if the message class
-  # implements the CompressedJsonable marker interface and the payload 
-  # is larger than this value. Only used for remote messages within 
-  # the cluster of the service.
-  compress-larger-than = 1024b
```

Comment:
A difference is that in Lagom it's decided per message by marking it with `Jsonable` or `CompressedJsonable`. That will still be supported in Lagom, or Akka users can easily setup same thing.

When deciding that a message should be compressed by marking it with `CompressedJsonable` it makes sense to compress almost all such messages apart from the smallest.

Akka's setting is for all messages, and since the overhead of gzip is high I don't want the default to be too aggressive. I tried these things out in https://github.com/akka/akka/pull/27067
Note that via akka/akka#27068 it will be possible to defined several bindings with different compression settings.

Reply:
see 52c2c14cefbc46777a13980297b7636c2c7e7665